build.sbt

```
@@ -6,6 +6,10 @@ val Versions = new {
   val logback = "1.4.5"
   val circe = "3.8.13"
   val scalaTest = "3.2.15"
+  val timepit = "0.10.2"
+  val tapirDoobie = "0.19.0-M7"
+  val postgresql = "42.5.4"
+  val doobie = "1.0.0-RC1"
```

Can be updated to RC2
```suggestion
  val doobie = "1.0.0-RC2"
```